pattern_import_export/tests/test_pattern_export.py

```diff
@@ -38,19 +36,18 @@ def test_get_header3(self):
         @return:
         """
         export_fields_m2m = self.ir_exports_m2m.export_fields.filtered(
```

Comment:
@kevinkhao it will be simplier to have demo data for profile and here update the profile line with the xmlid